sale_import_payment/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Import Payment",
+    "summary": "Adds payment capabilities to Sale Imports",
+    "version": "12.0.1.0.0",
+    "category": "Generic Modules/Sale",
+    "author": "Akretion",
+    "website": "https://github.com/akretion/sale-import",
+    "depends": ["sale_import_base", "payment"],
```

Comment:
we do not need to depend on payment as sale module already depend on it. https://github.com/OCA/OCB/blob/12.0/addons/sale/__manifest__.py#L12

I think this can be done in the sale_import mode as it do not necessite new dependency